spec/presenters/project_xml_presenter_spec.rb

```diff
@@ -524,7 +524,7 @@
   end
 
   describe "#globus_enable_approved?" do
-    it "returns false by default" do
```

Comment:
We don't have a "requested" vs "approved" workflow for these new fields yet (not part of the current Sprint) so I commented these tests.